crates/dslab-dag/src/runner.rs

```diff
@@ -143,6 +149,26 @@ impl DAGRunner {
         self.trace_log_enabled = flag;
     }
 
+    /// Returns DAG for this runner.
+    pub fn get_dag(&self) -> &DAG {
```

Comment:
Геттеры в Rust обычно называют без `get_`:

```suggestion
    pub fn dag(&self) -> &DAG {
```

Методы ниже аналогично.

Reply:
переименовал